content/en/blog/2019/isolated-clusters/index.md

```diff
@@ -0,0 +1,1828 @@
+clusters.
+For example, the same service can be called `withdraw` and `withdraw-funds`, and can reside in the `accounts` and
+`private-accounts` namespaces in different clusters. Such situation could happen in an organization where there is no
+uniform policy on naming of namespaces and services, or when the clusters belong to different organizations.
+- **expose-nothing by default**. None of the services in a cluster are exposed by default, the cluster owners must
+explicitly specify which services are exposed.
+- **boundary protection**. The access control of the traffic must be enforced at the ingress gateway, which stops
+forbidden traffic from entering the cluster. This requirement implements
+[Defense-in-depth principle](https://en.wikipedia.org/wiki/Defense_in_depth_(computing)) and is part of some compliance
+standards, such as the
+[Payment Card Industry (PCI) Data Security Standard](https://www.pcisecuritystandards.org/pci_security/).
+- **common trust may not exist**. The Istio sidecars in one cluster may not trust the Citadel certificates in other
+cluster, due to some security requirement or due to the fact that the cluster owners did not initially plan to couple
+the clusters.
+- **service location transparency**: consuming services send requests to the exposed services in remote clusters using
+local service names. The consuming services are oblivious to the fact that some of the destinations are in remote
+clusters and some are local services. The access is uniform, using the local service names of Kubernetes, for example `reviews.default.svc.cluster.local`.
+
+While **expose-nothing by default** and **boundary protection** are required to facilitate compliance and improve
+security, **non-uniform naming** and **common trust may not exist** are required when connecting
+clusters of different organizations, or of an organization that cannot enforce uniform naming or cannot or may not
+establish common trust between the clusters. **Service location transparency** is important since you do not want to
+change the code of your applications when the location of the consumed services changes.
+
+## The proposed implementation
+
+I propose to base connecting isolated clusters on the following principles:
+
+- use **standard Istio mechanisms** such as gateways, virtual services, destination rules, RBAC.
+- use **standard Istio installations**.
+- **ad hoc cluster _pairing_** at any time. The owners of the clusters can install Istio and operate it independently,
+  and decide to connect the clusters at some later point in time.
+- **private gateways for cross-cluster communication**, with dedicated certificates and private keys. Only the gateways
+  trust each other, there is no trust between sidecars from different clusters.
+
+In the following sections I demonstrate connecting isolated clusters using two clusters and the Istio
+[Bookinfo](/docs/examples/bookinfo/) application as an example.
+
+## Prerequisites
```

Comment:
@smawson PTAL at [this PR](https://github.com/istio/istio.io/pull/4788) which is trying to nail down these two terms in the glossary.

Reply:
@smawson I just tried to stay clear from providing any definitions.

Reply:
I want to avoid confusing readers. You are using the term cluster, and because you are not defining what it is, the reader has to guess. They may guess that this is a supported way to build a single mesh, and it is not. That's what I want to avoid by being explicit about exactly what this blog post is describing: An approach to link two meshes together to provide isolation.